static/views/edit-developer-profile.html

```diff
@@ -177,6 +250,60 @@ <h3>Edit Your Developer Profile</h3>
 			function update() {
 							sendReq('/x/udp');
 						}
+			window.onclick = function(event) {
+			  var modal = document.getElementById("developerMetadataModal");
+			  if (event.target == modal) {
+			    modal.style.display = "none";
+			  }
+			}
+
+			function openDeveloperMetadataModal(metadata_type, title, description, link, metadata_id) {
+				var modal = document.getElementById("developerMetadataModal");
+				if (metadata_type) {
+					document.getElementById("modalHeader").innerHTML = metadata_type;
+					document.getElementById("metadata_type").value = metadata_type;
+				}
+				modal.style.display = "block";
+				if (title) {
+					document.getElementById("title").value = title;
+				}
+				if (description) {
+					document.getElementById("description").value = description;
+				}
+				if (link) {
+					document.getElementById("link").value = link;
+				}
+				if (metadata_id) {
+					document.getElementById("metadata_id").value = metadata_id
+				}
+			}
+
+			function saveDeveloperMetadata() {
+			  var ID = document.getElementById("metadata_id").value;
+			  var metadata_type = document.getElementById("metadata_type").value;
```

Comment:
for consistency with the rest of the project we use snake-case for all our css/JS IDs, if you can please update this also would be great

Reply:
Done.